src/recommendation_service.py

```diff
@@ -86,5 +92,13 @@ def recommendation():
     return flask.jsonify(response_json), 200
 
 
+def log_it(func):
```

Comment:
Where is this decorator function used?

Reply:
future reference.

Reply:
What future reference are you planning?

Reply:
sir, i have an idea of wrapping any new function with this decorator. 
if this doesn't make sense.. i am okay to drop this

Reply:
Your idea is good. Maybe we should include this at the time of implementing this. But not with this PR.

Reply:
Please change accordingly in all other PR's as well.

Reply:
okay